pkg/provision/providers/qemu/node.go

```diff
@@ -174,7 +174,7 @@ func (p *provisioner) createNode(state *vm.State, clusterReq provision.ClusterRe
 
 		NanoCPUs: nodeReq.NanoCPUs,
 		Memory:   nodeReq.Memory,
-		DiskSize: nodeReq.DiskSize,
+		DiskSize: nodeReq.Disks[0].Size,
```

Comment:
I assume that we know for sure that `nodeReq.Disks` is not empty by the time we get here?

Reply:
yep. CreateDisks will fail otherwise